docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>Can you please place the ECG data into one of the comments of the PR and then link to that uploaded CSV?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>